app/decorators/group_decorator.rb

```diff
@@ -27,6 +27,21 @@ def possible_roles
     end
   end
 
+  def possible_roles_without_writing_permissions
+    excluded_permissions = [
```

Comment:
Wenn du das in ein class_attr auslagerst ist es in Wagons noch einfacher, das anzupassen

Reply:
Hätte es jetzt als Konstante `WritingPermissions` in role/types.rb definiert. Sollte auch einfach anpassbar sein im Wagon oder?